library/bignum_mod_raw.h

```diff
@@ -170,6 +170,31 @@ void mbedtls_mpi_mod_raw_sub( mbedtls_mpi_uint *X,
                               const mbedtls_mpi_uint *B,
                               const mbedtls_mpi_mod_modulus *N );
 
+/** \brief  Multiply two MPIs, returning the residue modulo the specified
+ *          modulus.
+ *
```

Comment:
Could clarify that this should only handle the case where the modulus `int_rep` is Montgomery.

Reply:
This has been done, so marking resolved